azkaban-web-server/src/web/js/azkaban/util/flow-loader.js

```diff
@@ -161,13 +184,13 @@ var nodeClickCallback = function (event, model, node) {
           model.trigger("centerNode", node)
         }
       }
-    ];
+    ]);
   }
   contextMenuView.show(event, menu);
 }
 
 var jobClickCallback = function (event, model, node) {
-  console.log("Node clicked callback");
+  console.log("Job clicked callback");
```

Comment:
By the way, the content of `jobClickCallback` seems pretty much the same as `nodeClickCallback`'s. But I don't even know what it's for? All cases that I tested about this PR worked correctly even though I didn't "echo" the changes into `jobClickCallback`.

Reply:
if it's not being used anywhere, we should delete it.